uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-lucene/src/test/java/org/uberfire/ext/metadata/backend/lucene/index/LuceneInedexManagerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.uberfire.ext.metadata.backend.lucene.index;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.uberfire.ext.metadata.model.KCluster;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LuceneInedexManagerTest {
```

Comment:
@akumar074 in another PR (could be whenever you do another change), could you please fix the name of this class?